build/scripts/PullRequestValidation/ValidateIssuesForPullRequest.ps1

```diff
@@ -0,0 +1,47 @@
+using module ..\GitHub\GitHubPullRequest.class.psm1
+using module ..\GitHub\GitHubIssue.class.psm1
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string] $Repository
+    )
+
+# Set error action
+$ErrorActionPreference = "Stop"
+
+Write-Host "Validating PR $PullRequestNumber"
+
+$pullRequest = [GitHubPullRequest]::Get($PullRequestNumber, $Repository)
+$prDescription = $pullRequest.GetBody()
+$issuesStartingPoint = "Fixes #"
+
+$issueStartingIndex = -1
+if($prDescription) {
```

Comment:
Consider doing the opposite check earlier on. `if (-not $prDescription) { throw "" }`. Then you don't need to set $issueStartingIndex = -1. IndexOf will return -1 anyway if it doesn't find anything.

Reply:
Potato, potato (potayto, potahto) 😆

Reply:
Done